Program.cs

```diff
@@ -182,6 +182,21 @@ public static void Main(string[] args)
                     // package for distribution
                     runCommand("ditto", $"-ck --rsrc --keepParent --sequesterRsrc {stagingPath}/osu!.app {releasesPath}/osu!.app.zip");
                     break;
+                case RuntimeInfo.Platform.Linux:
+
+                    // need to add --self-contained flag for AppImage distribution.
+                    runCommand("dotnet", $"publish -r linux-x64 --self-contained --configuration Release -o {stagingPath}/osu.AppDir  /p:Version={version}");
+
+
+                    runCommand("chmod", $"-R 755 {stagingPath}/osu.AppDir/*");
+
+
+                    runCommand("rsync", $"-av --progress AppDir/ {stagingPath}/osu.AppDir");
+
+                    // requirement to have the linuxdeploy script to use this.
+                    // See: https://github.com/linuxdeploy/linuxdeploy

```

Comment:
From their repo it's look like it's native application, not script.
Consider to move such comment to readme.md.